provider/datagen/src/transform/cldr/decimal/mod.rs

```diff
@@ -107,21 +110,23 @@ impl IterableDataProvider<DecimalSymbolsV1Marker> for NumbersProvider {
     }
 }
 
-impl TryFrom<&cldr_serde::numbers::Numbers> for DecimalSymbolsV1<'static> {
+struct NumbersWithNumsys<'a>(pub &'a cldr_serde::numbers::Numbers, pub TinyAsciiStr<8>);
```

Comment:
suggestion (nb): let's use inherent functions and avoid TryFrom in these cases

(in general i find that our TryFrom impls in transform tend to bite us on changes like this)

Reply:
Acknowledged. I'd like to keep the diff small. In the mean time, this is all private code.